doc/classes/Variant.xml

```diff
@@ -5,20 +5,26 @@
 	</brief_description>
 	<description>
 		In computer programming, a Variant class is a class that is designed to store a variety of other types. Dynamic programming languages like PHP, Lua, JavaScript and GDScript like to use them to store variables' data on the backend. With these Variants, properties are able to change value types freely.
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var foo = 2 # foo is dynamically an integer
 		foo = "Now foo is a string!"
 		foo = Reference.new() # foo is an Object
 		var bar: int = 2 # bar is a statically typed integer.
 		# bar = "Uh oh! I can't make static variables become a different type!"
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		// C# is statically typed, but uses the Mono [code]object[/code] type.
```

Comment:
We can be more descriptive here, including examples of what is expected as the normal behavior first (static types), and then we can give an example with using object.

Reply:
See new try. Oh you mean by example something like a print. I don't really have an idea